examples/footer/index.html

```diff
@@ -8,59 +8,85 @@
+              <li>
+                <a href="https://www.stjude.org/about-st-jude.html">About</a>
+              </li>
+              <li>
+                <a href="https://stjude.org/contact-us.html">Contact</a>
+              </li>
+            </ul>
+          </div>
+          <div class="col-md-3 col-6 offset-xl-0 offset-md-2 footer-apps">
+            <h6>St. Jude Cloud Apps</h6>
+            <ul>
+              <li>
+                <a href="https://cstn.stjude.cloud">CSTN</a>
+              </li>
+              <li>
+                <a href="https://platform.stjude.cloud">Platform</a>
+              </li>
+              <li>
+                <a href="https://viz.stjude.cloud">Visualization Community</a>
+              </li>
+            </ul>
+          </div>
+          <div class="col-xl-3 col-lg-6 col-8 offset-xl-0 offset-lg-3 offset-2 footer-social">
+            <h6>Follow Us</h6>
+            <div class="d-flex justify-content-between">
+                <a target="_blank" rel="noopener noreferrer" href="https://www.facebook.com/stjuderesearch" class="fab fa-facebook"></a>
+
+                <a target="_blank" rel="noopener noreferrer" href="https://twitter.com/StJudeResearch" class="fab fa-twitter"></a>
+
+                <a target="_blank" rel="noopener noreferrer" href="https://www.instagram.com/stjuderesearch/" class="fab fa-instagram"></a>
+
+                <a target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/company/st-jude-childrens-research-hospital/" class="fab fa-linkedin"></a>
+
+                <a target="_blank" rel="noopener noreferrer" href="https://www.youtube.com/user/MyStJude" class="fab fa-youtube"></a>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="sub-footer container-fluid">
+        <div class="container">
+          <div class="row">
+            <div class="col-xl-6 col-lg-12 order-last">
+              <ul>
+                <li>
+                  <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/u-s-privacy-policy-statement.html" title="U.S. Privacy Policy Statement">Privacy Policy</a> <span>&#8226;</span>
+                </li>
+                <li>
+                  <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal.html" title="Disclaimer / Registrations / Copyright Statement">Disclaimer, Registrations, and Copyright</a><span>&#8226;</span>
+                </li>
+                <li>
+                  <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-privacy-practices.html" title="Notice of Privacy Practices (HIPAA)">Privacy (HIPAA)</a><span>&#8226;</span>
+                </li>
+                <li>
+                  <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-non-discrimination.html" title="Notice of Non-Discrimination">Non-Discrimination</a>
+                </li>
+              </ul>
```

Comment:
I'm not sure if it's intended, but the list bullets are showing on large displays. At the very least, the spacing should be uniform between each item.

<img width="560" alt="Screen Shot 2020-03-27 at 5 15 50 PM" src="https://user-images.githubusercontent.com/2433649/77805534-1cbc0480-7050-11ea-83b1-4828b82a365e.png">

Additionally, the text color changes between sizing breakpoints:

<img width="577" alt="Screen Shot 2020-03-27 at 5 15 59 PM" src="https://user-images.githubusercontent.com/2433649/77805554-2a718a00-7050-11ea-9264-95b9cfd5fc35.png">

Reply:
I like the bullets as delimiters but agree to equal space (and I'd say at least that of what's to the right of each bullet)

Reply:
I think it's definitely nitpicky, but I prefer the spacing from the institutional footer over the bullets.

![image](https://user-images.githubusercontent.com/48360246/77930607-b5c96600-7270-11ea-9d87-af83b7bf22b0.png)

Reply:
I've fixed the spacing of the bullet points, and changed the links to remain the same color on all sizes.